registry/src/pages/register.js

```diff
@@ -98,8 +100,14 @@ const Register = () => {
           <Link to="/account/forgot-password"> Forgot password </Link>
         </p>
       </form>
+    </Container> ) : (
```

Comment:
Just make the text inside "SignUp button" change to loading indicator instead of the complete page.

Reply:
sure.